### ISSUE TITLE
typings install error

### ISSUE THREAD

**Issue by Sfairet** [nkalinov/ng2-datetime#58]
Hello!

I try install ng2-datetime in production mode, but postinstall script failed.
```
npm ERR! ng2-datetime@1.2.0 postinstall: `typings install`
npm ERR! Exit status 1
```
I havnt installed in production typings, it available only in devDependencies.
If i install typings manually all work fine, but typings should not be present at the production server.

**Comment by nkalinov** [comment]
Hi @Sfairet,
Normally you would not install packages in production neither, instead had built/bundled everything before, but hey, that's your way.
Still, I'll remove the postinstall hook and let people install or not them manually.

**Comment by nkalinov** [comment]
Fixed in 1.2.1

**Comment by Sfairet** [comment]
@nkalinov thank you!
